encoder/vaapiencoder_hevc.cpp

```diff
@@ -1022,16 +1023,7 @@ Encode_Status VaapiEncoderHEVC::setParameters(VideoParamConfigType type, Yami_PT
     case VideoParamsTypeAVC: {
```

Comment:
I think it should be ok for hevc use avc structs, because hevc is evolution of avc. But Avc should not reuse hevc structs.